app/views/schools/cohorts/show.html.erb

```diff
@@ -17,10 +17,20 @@
         </h2>
         <ul class="app-task-list__items">
 
+          <li class="app-task-list__item">
+            <span class="app-task-list__task-name">
+              <%= govuk_link_to "Choose the type of training",
+                edit_schools_cohort_programme_path(@cohort.start_year) %>
+            </span>
+
+            <%= render AutoTagComponent.new(text: "done") %>
```

Comment:
We aren't using css to automatically transform these into uppercase so I'd just Done rather than done

Reply:
Actually @amazimbe that's changing back - breaking news. Tags will be shouty case again